### ISSUE TITLE
[wasm] Handle multiple python directories in emsdk

### ISSUE THREAD

**Issue by radekdoulik** [dotnet/runtime#51746]
Sometime emsdk can contain multiple versions of python or other tools. We use `$([System.IO.Directory]::GetDirectories($(_PythonToolsBasePath)))` to get the `_PythonToolsVersionedPath` and assume there is only one subdirectory present.

    src\mono\wasm\build\WasmApp.targets(289,5): error : Specified Emscripten sdk at $(EMSDK_PATH)=C:\Users\rodo\git\emsdk\ is missing some paths: $(EmscriptenPythonToolsPath)=...\emsdk\python\3.7.4-pywin32_64bit;...\emsdk\python\3.9.2-1_64bit\ . Emscripten SDK is required for AOT'ing assemblies.

**Comment by lewing** [comment]
Yeah, I think we can improve how we handle the case when EMSDK_PATH is set.
